src/main/resources/com/mathworks/ci/StartupOptions/help-startupOptions.html

```diff
@@ -0,0 +1,3 @@
+<div>
+ A space-separated list of MATLAB startup options.
```

Comment:
Should we have link to showup the actual option list ? How one would learn about these options

Reply:
Good question, I wonder if @mw-hrastega has a preference here?

Reply:
@sameagen-MW: I think we should be consistent with how other options are represented in the software. If others do not include links, we cannot provide a link here.

Reply:
That makes sense to me!